src/components/ChatContent/ChatContent.js

```diff
@@ -270,14 +285,19 @@ export default class ChatContent extends React.Component {
         this.addResponseMessage({
           content: (
             <span>
-              Looks like you’re trying to query a Microsoft Dynamics data source.
-              <a href={response.data.data.authorization_url} target='_blank' rel="noreferrer">
+              Looks like you’re trying to query a Microsoft Dynamics data
+              source.
+              <a
+                href={response.data.data.authorization_url}
+                target="_blank"
+                rel="noreferrer"
+              >
                 Click here to authorize access then try querying again.
               </a>
             </span>
           ),
         })
-      } else {
+      } else if (_get(response?.data, 'message') !== 'Request cancelled') {
```

Comment:
Can you use optional chaining instead? My goal is to eventually get rid of all _get functions eventually